### PR TITLE
refactor(ui): refactor default logo

### PR DESCRIPTION
**Pull request by simeng-li**
## Summary

Refactor default logo logic

1. remove the fallback locale static logo under UI
2. add default logo url in seed
3. add logo non-empty string validation in sign-in-exp guard
4. add darkLogo to logo fallback logic in UI. should display light mode logo, if no darkmode logo provided. 

image

## Testing

test locally
@logto-io/eng

### CODE REVIEW

File: packages/phrases/src/locales/zh-cn.ts [hunk 1]
```diff
@@ -583,6 +583,7 @@ const errors = {
   },
   sign_in_experiences: {
     empty_content_url_of_terms_of_use: '你启用了“使用条款”，请添加使用条款 URL。',
+    empty_logo: '请输入一个 App logo',
```
Comment (gao-sun) [inline packages/phrases/src/locales/zh-cn.ts:586]:
```suggestion
    empty_logo: '请提供 App logo URL',
```

File: packages/phrases/src/locales/en.ts [hunk 2]
```diff
@@ -603,6 +603,7 @@ const errors = {
   sign_in_experiences: {
     empty_content_url_of_terms_of_use:
       'Empty "Terms of use" content URL. Please add the content URL if "Terms of use" is enabled.',
+    empty_logo: 'Empty Logo. Please add a branding logo',
```
Comment (gao-sun) [inline packages/phrases/src/locales/en.ts:606]:
```suggestion
    empty_logo: 'Please enter your logo URL',
```

File: packages/schemas/src/seeds/sign-in-experience.ts [hunk 3]
```diff
@@ -12,7 +12,7 @@ export const defaultSignInExperience: Readonly<CreateSignInExperience> = {
   },
   branding: {
     style: BrandingStyle.Logo,
-    logoUrl: '',
+    logoUrl: 'https://raw.githubusercontent.com/logto-io/logto/master/logo.png',
```
Comment (gao-sun) [inline packages/schemas/src/seeds/sign-in-experience.ts:15]:
```suggestion
    logoUrl: 'https://logto.io/logo.svg',
```

File: packages/schemas/src/seeds/sign-in-experience.ts [hunk 4]
```diff
@@ -37,7 +37,7 @@ export const adminConsoleSignInExperience: CreateSignInExperience = {
   ...defaultSignInExperience,
   branding: {
     style: BrandingStyle.Logo_Slogan,
-    logoUrl: '',
+    logoUrl: 'https://raw.githubusercontent.com/logto-io/logto/master/logo.png',
```
Comment (gao-sun) [inline packages/schemas/src/seeds/sign-in-experience.ts:40]:
```suggestion
    logoUrl: 'https://logto.io/logo.svg',
```

**Review by gao-sun — APPROVED** [review]

File: packages/phrases/src/locales/en.ts (hunk 2 above)
Comment (simeng-li) [reply packages/phrases/src/locales/en.ts]: updated

File: packages/schemas/src/seeds/sign-in-experience.ts (hunk 3 above)
Comment (simeng-li) [reply packages/schemas/src/seeds/sign-in-experience.ts]: updated

File: packages/schemas/src/seeds/sign-in-experience.ts (hunk 4 above)
Comment (simeng-li) [reply packages/schemas/src/seeds/sign-in-experience.ts]: updated

**Pull request merged**